frontend/src/components/Checkbox/CheckboxOthers.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChangeEvent, cloneElement, isValidElement, useRef } from 'react'
```

Comment:
We can’t do `Checkbox.Others` because `Checkbox` is of type `ComponentWithAs<CheckboxProps, ‘input’> (because we use `forwardRef`) I found a suggested solution to introduce a type `ComponentWithAs<CheckboxProps, ‘input’> & {Others?: typeof Others}` but this doesn’t really work because `Checkbox.Others` could be undefined. I think a workaround is to export an object `{ CheckboxOption: Checkbox, CheckboxOthers: Others }`, which we can then use as `Checkbox.CheckboxOption` and `Checkbox.CheckboxOthers`. Wdyt?

Reply:
Possible to sidestep with a typescript cast: https://stackoverflow.com/a/63454836